migrations/v2/V202408150822__v2.3.x.sql

```diff
@@ -0,0 +1,10 @@
+--In the Entities table, Set AllowCreateAPI to 1 for a few integration related Entities
+UPDATE __mj.Entity
+SET AllowCreateAPI = 1
+WHERE ID IN (
```

Comment:
@JS-BC  this is okay but remember ID values are now persistent/durable across time/systems, so you should just use an `ID IN ('z','y','x')` type of statement (you could put comments in there on which ID is which entity)